api-server/api-server-common/src/storage/impls/postgres/queries.rs

```diff
@@ -142,83 +149,91 @@ impl<'a, 'b> QueryFromConnection<'a, 'b> {
                     SET value = $2;",
                 &[&(block_height, block_id).encode()],
             )
+            .await
             .map_err(|e| ApiServerStorageError::LowLevelStorageError(e.to_string()))?;
 
         Ok(())
     }
 
-    fn just_execute(&mut self, query: &str) -> Result<(), ApiServerStorageError> {
+    async fn just_execute(&mut self, query: &str) -> Result<(), ApiServerStorageError> {
         self.tx
             .execute(query, &[])
+            .await
             .map_err(|e| ApiServerStorageError::LowLevelStorageError(e.to_string()))?;
 
         Ok(())
     }
 
-    pub fn create_tables(&mut self) -> Result<(), ApiServerStorageError> {
+    pub async fn create_tables(&mut self) -> Result<(), ApiServerStorageError> {
         logging::log::info!("Creating database tables");
 
         self.just_execute(
             "CREATE TABLE ml_misc_data (
             name TEXT PRIMARY KEY,
             value BLOB NOT NULL
```

Comment:
I might change these `BLOB`s to `bytea` or `TEXT`

Reply:
Feel free to use whatever works for you.